### PR TITLE
add event to log sb install

### PR DESCRIPTION
**Pull request by winkerVSbecks**

### CODE REVIEW

File: src/components/screens/DocsScreen/FeatureSnippets.tsx [hunk 1]
```diff
@@ -13,14 +20,21 @@ export function PureFeatureSnippets({ framework, snippetsByFramework }) {
...
+    </div>
+  ) : (
     <div className="aside">
       <Snippet />
     </div>
   );
 }
 
 export function FeatureSnippets({ currentFramework, paths }) {
+  const isInstallSnippet = paths[0].includes(INSTALL_PATH);
```
Comment (kylegach) [inline src/components/screens/DocsScreen/FeatureSnippets.tsx:36]: Given that FeatureSnippets is used directly within MDX, explicitly passing an `isInstallSnippet` prop directly to the component seems less finicky to me.

File: src/components/screens/DocsScreen/FeatureSnippets.tsx [hunk 2]
```diff
@@ -13,14 +20,21 @@ export function PureFeatureSnippets({ framework, snippetsByFramework }) {
     }
   }
 
-  return (
+  return isInstallSnippet ? (
+    // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions
+    <div className="aside" onClick={() => logInstall(framework)}>
```
Comment (kylegach) [inline src/components/screens/DocsScreen/FeatureSnippets.tsx:25]: This approach won't catch people who operate the Copy button with their keyboard. But that's likely done seldomly?

File: src/components/screens/DocsScreen/FeatureSnippets.tsx (hunk 1 above)
Comment (winkerVSbecks) [reply src/components/screens/DocsScreen/FeatureSnippets.tsx:36]: 🤔 will that work for older doc versions too?

Comment (kylegach) [reply src/components/screens/DocsScreen/FeatureSnippets.tsx:36]: Alternatively, you could go with a different abstraction and pass the logged event (e.g. `event=""install_storybook"`) to FeatureSnippets in the MDX content.

Comment (jonniebigodes) [reply src/components/screens/DocsScreen/FeatureSnippets.tsx:36]: I agree with @kylegach 's reasoning on this.

Comment (kylegach) [reply src/components/screens/DocsScreen/FeatureSnippets.tsx:36]: (Sorry, I keep thinking of ideas.)

As a final alternative, you could log something `\`${framework}, ${paths[0]}\`` for _every_ FeatureSnippet (and CodeSnippet?). Then we could map paths to outcomes after we collect the data.

**Comment by winkerVSbecks** [comment]
@kylegach updated to log all snippets

**Pull request merged**